torch/_dynamo/variables/base.py

```diff
@@ -452,6 +452,14 @@ def next_variable(self, tx):
     def is_strict_mode(self, tx):
         return tx.strict_checks_fn and tx.strict_checks_fn(self)
 
+    def is_mutable(self):
+        """Whether Dynamo allows mutation on this variable."""
+        return self.mutation_type
```

Comment:
Nit - `return self.mutation_type is not None`?

Reply:
Good call, updating.